src/ol/geom/geometry.js

```diff
@@ -150,6 +151,7 @@ ol.geom.Geometry.prototype.computeExtent = goog.abstractMethod;
  * @param {number} x X.
  * @param {number} y Y.
  * @return {boolean} Contains (x, y).
+ * @api
```

Comment:
Are you working with an application where it is a performance hit to use `containsCoordinate`?